schemas/test/en/test_anchor_scrolling.json

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
I'm not sure about the schema name. Maybe something like `test_submit_with_summary_return_to_anchor`